linera-core/src/worker.rs

```diff
@@ -92,6 +92,15 @@ static NUM_BLOCKS: LazyLock<IntCounterVec> = LazyLock::new(|| {
     register_int_counter_vec("num_blocks", "Number of blocks added to chains", &[])
 });
 
+#[cfg(with_metrics)]
+static CERTIFICATES_SIGNED: LazyLock<IntCounterVec> = LazyLock::new(|| {
+    register_int_counter_vec(
+        "certificates_signed",
+        "Number of certificates signed by each validator",
```

Comment:
(We could be more specific, since we're not counting timeouts or validated blocks.)

```suggestion
        "Number of confirmed block certificates signed by each validator",
```

Reply:
My bad, completely forgot to address this: https://github.com/linera-io/linera-protocol/pull/3058